### DISCUSSION TITLE
Where is the discord server mentioned on the wiki

### DISCUSSION THREAD

**Discussion by JustSimplyKyle** [hyprwm/Hyprland#432] [Q&A]
Sorry but, I can seem to find where is it.
https://github.com/hyprwm/Hyprland/wiki/Installation#foreword
Or does that line just means you can ask for help, but no support provided by the hyprland team?

**Comment by vaxerski** [comment]
![image](https://user-images.githubusercontent.com/43317083/180966590-5f8278ca-e19a-43d2-a0f7-03ab50d2bef2.png)

**Reply by JustSimplyKyle** [reply]
Ohhh, I should've look at the main page bruh
Maybe make the Discord server mentioned in the wiki an actual hyprlink to it?

**Comment by bound-variable** [comment]
I also couldn't find it.

**Reply by vaxerski** [reply]
dam yall blind, made it a hyperlink.
